start_bs_to_kafka_cmd.bat

```diff
@@ -9,5 +9,13 @@ set EPICS_CAS_INTF_ADDR_LIST=127.0.0.1
 set EPICS_CAS_BEACON_ADDR_LIST=127.255.255.255
 
 set PYTHONUNBUFFERED=TRUE
+
+
+if "%INSTRUMENT%" == "NDXHIFI" (
```

Comment:
Are you sure about this having the `NDX` prefix?

After config_env on hifi I get:

```
> set INSTRUM
INSTRUMENT=HIFI
```

Reply:
i should have checked that on HIFI not my own machine. just updated...